src/App.css

```diff
@@ -33,10 +33,40 @@
   }
 }
 
-.card {
+ .card {
   padding: 2em;
 }
 
 .read-the-docs {
   color: #888;
 }
+
+
+.cars-container {
+  display: flex;
```

Comment:
Please, consider about more specific naming, for example `.car-form-container`

Reply:
fixed and changed to ".car-form-container"